Version 2.1.0 (#443)
GeneriekPublicatiePlatformWoo/registratie-component: CHANGELOG.rst

Release notes
=============

2.1.0 (2026-04-22)
==================

The release candidate is now released as stable version.

Bugfixes
--------

> * [:issue:`435`] microsoft files no longer get corrupted while stripping the metadata

Reviewer: ```suggestion
* [:issue:`435`] Fixed MS Office files being corrupted by the metadata stripping.
```
bugfix entries pretty much always should start with "Fixed ..."

> * [:issue:`433`] ensure that the download api endpoint works for older documents

Reviewer: ```suggestion
* [:issue:`433`] Fixed the download endpoint for legacy documents created before the metadata stripping feature was introduced.
```
Also make sure that changelog entries start with a capital letter and end with a period.

  which aren't stripped (but are eligible per 2.1.0-rc.0)

> * Fixed broken asset references

Reviewer: ```suggestion
* Fixed broken asset references.
```
